test/Feature/InsertSpec.hs

```diff
@@ -293,29 +293,30 @@ spec actualPgVersion = do
         -- reset pk sequence first to make test repeatable
         request methodPost "/rpc/reset_sequence"
             [("Prefer", "tx=commit")]
-            [json|{"name": "items_id_seq", "value": 20}|]
+            [json|{"name": "items2_id_seq", "value": 20}|]
           `shouldRespondWith`
             [json|""|]
 
-        request methodPost "/items" [("Prefer", "return=representation")] "{}"
-          `shouldRespondWith` [json|[{ id: 20 }]|]
-          { matchStatus  = 201,
-            matchHeaders = []
-          }
+        request methodPost "/items2"
```

Comment:
Because of concurrency issues, we're running all the insert tests that use the auto incrementing PK on `items` on different tables now: items, items2 and items3. No conflict anymore.